src/Orleans.Core.Abstractions/Placement/StatelessWorkerPlacement.cs

```diff
@@ -9,7 +9,7 @@ internal class StatelessWorkerPlacement : PlacementStrategy
 
         public int MaxLocal { get; private set; }
 
-        internal StatelessWorkerPlacement(int maxLocal = -1)
+        public StatelessWorkerPlacement(int maxLocal = -1)
```

Comment:
Maybe we should add a default constructor instead and make this argument non-optional on this ctor